File: Java/PetShop.java
public class PetShop {
    // Attributes
    private String id;
    private String namaProduk;
    private String kategoriProduk;
    private int hargaProduk;
    private boolean fotoProduk;

    // Constructor
    public PetShop(String id, String namaProduk, String kategoriProduk, int hargaProduk, boolean fotoProduk) {
        this.id = id;
        this.namaProduk = namaProduk;
        this.kategoriProduk = kategoriProduk;
        this.hargaProduk = hargaProduk;
        this.fotoProduk = fotoProduk;
    }

    // Setter and Getter
    public void setId(String id) {
        this.id = id;
    }

    public void setNamaProduk(String namaProduk) {
        this.namaProduk = namaProduk;
    }

    public void setKategoriProduk(String kategoriProduk) {
        this.kategoriProduk = kategoriProduk;
    }

    public void setHargaProduk(int hargaProduk) {
        this.hargaProduk = hargaProduk;
    }

    public void setFotoProduk(boolean fotoProduk) {
        this.fotoProduk = fotoProduk;
    }

    // Getters
    public String getId() {
        return this.id;
    }

    public String getNamaProduk() {
        return this.namaProduk;
    }

    public String getKategoriProduk() {
        return this.kategoriProduk;
    }

    public int getHargaProduk() {
        return this.hargaProduk;
    }

    public boolean getFotoProduk() {
        return this.fotoProduk;
    }

    // Methods
    public void showProduct() {
        System.out.println("| ID Produk : " + this.id);
        System.out.println("| Nama Produk : " + this.namaProduk);
        System.out.println("| Kategori Produk : " + this.kategoriProduk);
        System.out.println("| Harga Produk : " + this.hargaProduk);
        System.out.println("| Foto Produk : " + (this.fotoProduk ? "Ada" : "Tidak Ada"));
        System.out.println("=======================================");
    }

    public void editProduct(String namaProduk, String kategoriProduk, int hargaProduk, boolean fotoProduk) {
        this.namaProduk = namaProduk;
        this.kategoriProduk = kategoriProduk;
        this.hargaProduk = hargaProduk;
        this.fotoProduk = fotoProduk;
    }
}

File: Python/PetShop.py
class PetShop:

    # Attributes
    __id = ""
    __nama_produk = ""
    __kategori_produk = ""
    __harga_produk = 0
    __foto_produk = False

    # Constructor
    def __init__(this, id, nama_produk, kategori_produk, harga_produk, foto_produk):
        this.__id = id
        this.__nama_produk = nama_produk
        this.__kategori_produk = kategori_produk
        this.__harga_produk = harga_produk
        this.__foto_produk = foto_produk

    # Setter and Getter methods
    # Setters
    def set_id(this, id):
        this.__id = id
    
    def set_nama_produk(this, nama_produk):
        this.__nama_produk = nama_produk
    
    def set_kategori_produk(this, kategori_produk):
        this.__kategori_produk = kategori_produk
    
    def set_harga_produk(this, harga_produk):
        this.__harga_produk = harga_produk
    
    def set_foto_produk(this, foto_produk):
        this.__foto_produk = foto_produk
    
    # Getters
    def get_id(this):
        return this.__id
    
    def get_nama_produk(this):
        return this.__nama_produk
    
    def get_kategori_produk(this):
        return this.__kategori_produk
    
    def get_harga_produk(this):
        return this.__harga_produk
    
    def get_foto_produk(this):
        return this.__foto_produk
    
    # Methods
    def show_product(this):
        print("| ID Produk :", this.__id)
        print("| Nama Produk :", this.__nama_produk)
        print("| Kategori Produk :", this.__kategori_produk)
        print("| Harga Produk :", this.__harga_produk)
        print("| Foto Produk :", "Ada" if this.__foto_produk else "Tidak Ada")
        print("=======================================")
    
    def edit_product(this, nama_produk, kategori_produk, harga_produk, foto_produk):
        this.__nama_produk = nama_produk
        this.__kategori_produk = kategori_produk
        this.__harga_produk = harga_produk
        this.__foto_produk = foto_produk